source/wp-content/themes/wporg-showcase-2022/src/site-screenshot/index.php

```diff
@@ -45,18 +45,19 @@ function render( $attributes, $content, $block ) {
 	$post_ID = $block->context['postId'];
 	$post = get_post( $post_ID );
 
-	$screenshot = site_screenshot_src( $post );
-
-	if ( isset( $attributes['useHiRes'] ) && true === $attributes['useHiRes'] ) {
-		$screenshot = add_query_arg( 'scale', 2, $screenshot );
-	}
+	$screenshot = get_site_screenshot_src( $post, $attributes['type'] );
 
 	$loading = 'eager';
 	if ( isset( $attributes['lazyLoad'] ) && true === $attributes['lazyLoad'] ) {
 		$loading = 'lazy';
 	}
 
-	$img_content = "<img src='" . esc_url( $screenshot ) . "' alt='" . the_title_attribute( array( 'echo' => false ) ) . "' loading='" . esc_attr( $loading ) . "' />";
+	$img_content = sprintf(
+		'<img src="%1$s" alt="%2$s" loading="%3$s" />',
+		esc_url( $screenshot ),
+		the_title_attribute( array( 'echo' => false ) ),
```

Comment:
Can we change this to use the image alt text?

Reply:
This image could be an mshots image, and we don't really know anything about it at this point since `get_site_screenshot_src` only returns a URL. But I do want to revisit how these are linked/alt-text'd in the future.

Reply:
(I'm going to hold off on creating an issue for that until we have the final design and know how these screenshots will be used)